tools/downloader/README.md

```diff
@@ -81,7 +88,8 @@ The basic usage is to run the script like this:
 ```
 
 This will convert all models into the Inference Engine IR format. Models that
-were originally in that format are ignored.
+were originally in that format are ignored. If model in PyTorch's format they will be 
```

Comment:
You have mismatched tenses here: singular "model" in the if clause and plural "they" in the main clause. Also, there's no verb in the if clause.

Reply:
You fixed the first error, but not the second one.

Reply:
Fixed. Resolve?